system-addon/content-src/asrouter/asrouter-content.jsx

```diff
@@ -47,15 +53,15 @@ export class ASRouterUISurface extends React.PureComponent {
     this.onMessageFromParent = this.onMessageFromParent.bind(this);
     this.sendImpression = this.sendImpression.bind(this);
     this.sendUserActionTelemetry = this.sendUserActionTelemetry.bind(this);
-    this.state = {message: {}};
+    this.state = {message: {}, bundle: {}};
   }
 
-  sendUserActionTelemetry(extraProps = {}) {
-    const {message} = this.state;
-    const eventType =  `${message.provider}_user_event`;
+  sendUserActionTelemetry(extraProps = {}, bundledMessageID) {
```

Comment:
I think it might be better to remove the`bundleMessageId` param and just add`message_id` to `extraProps` when you are calling the function in onboarding messages, i.e. 

OnboardingMessage.jsx:
```js
this.props.sendUserActionTelemetry({message_id: this.props.id});
```

Additionally, you could do something like this here:
```js
if (!message && !extraProps.message_id) {
  throw new Error(`You must provider a message_id for bundled messages...`);
}